cmd/root.go

```diff
@@ -118,8 +118,9 @@ or ./scorecard --{npm,pypi,rubgems}=<package_name> [--checks=check1,...] [--show
 		enabledChecks := checker.CheckNameToFnMap{}
 		if len(checksToRun) != 0 {
 			for _, checkToRun := range checksToRun {
-				if checkFn, ok := checks.AllChecks[checkToRun]; ok {
-					enabledChecks[checkToRun] = checkFn
+				checkIndex := strings.ToLower(checkToRun)
```

Comment:
We can update this code to:

```
for _, checkToRun := range checksToRun {
  for key, checkFn := range checks.AllChecks {
    if strings.ToLower(checkToRun) == strings.ToLower(key) {
     enabledChecks[key] = checkFn
   }
  }  
}
```

Then the rest of the code can remain the same.

Reply:
Thanks @azeemshaikh38. I have made the above change and removed the need to index check functions with the lower case key so that constants like `checks.CheckBranchProtection` remain to work if we need to delete functions from the map, as in the case https://github.com/ossf/scorecard/blob/main/cron/worker/main.go#L183